ssntp/client.go

```diff
@@ -487,6 +487,7 @@ func (client *Client) UUID() string {
 // status frame or a CONFIGURE command one.
 func (client *Client) ClusterConfiguration() (payloads.Configure, error) {
 	var conf payloads.Configure
+	conf.Init()
```

Comment:
Yeah, agree with that, I just talked with @mrkz, @sameo should we keep that fillDefaults function or use the one at payloads/configure.go ?